handlers/sql_generic_v3/readme.md

```diff
@@ -0,0 +1,81 @@
+# Sql Generic V3
+
+The generic sql handler uses a predefined query to execute against a SQL database.  The predefined queries are kept in a Space Form submission and retrieved when the handler runs.  The Space Form's slug is defined as a info value for the handler.  
+In order for the handler to retrieve the correct query the Space Form needs a unique index to search on.  The handler expects the Space From to have a field with the name **Template Name** and the form should use that field as a unique index.
+
+## Info values
+[database_server]
+
+	JDBC connection that specifies the database location.

```

Comment:
This description is confusing. Maybe change it to be something like the DNS name or IP address of the database server.